mllam_data_prep/create_dataset.py

```diff
@@ -143,6 +151,10 @@ def create_dataset(config: Config):
             dataset_name=dataset_name,
         )
 
+        # TODO: remove this once grid_mapping is set in config file
+        for var in ds.data_vars:
+            ds[var].attrs["grid_mapping"] = "__common__"
+
```

Comment:
@leifdenby this is just a remark for the future when we allow the setting of variable specific projections or read the projection information directly from the input file

Reply:
We can delete this and mention it maybe in an issue so we do not forget.